starter-code/ShoppingListWithDetailView/app/src/main/java/ly/generalassemb/drewmahrt/shoppinglistdetailview/SecondFragment.java

```diff
@@ -0,0 +1,49 @@
+package ly.generalassemb.drewmahrt.shoppinglistdetailview;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+/**
+ * Created by Serkan on 01/11/16.
+ */
+
+public class SecondFragment extends Fragment {
```

Comment:
I suggest calling this DetailFragment rather than SecondFragment, to make it more clear what feature it is part of.